msp430-gcc.rb

```diff
@@ -30,7 +30,8 @@ def install
     # which, upon further inspection, arises when xgcc bails out when it sees
     # this argument.
     ENV.remove_from_cflags '-Qunused-arguments'
-    ENV.remove_from_cflags '-march=native'
+    ENV.remove_from_cflags(/ ?-march=\S*/)
+    ENV.remove_from_cflags(/ ?-msse4(\.\d)?/)
```

Comment:
I'm not aware of which flags homebrew sets. Looking at [hardware.rb](https://github.com/Homebrew/homebrew/blob/5a2292ff22d08b6c8b7c06bb15d6812577c977d2/Library/Homebrew/os/mac/hardware.rb#L4) maybe also the `-mcpu` can be removed.